components/financial_assistance/app/controllers/financial_assistance/applications_controller.rb

```diff
@@ -8,7 +8,7 @@ class ApplicationsController < FinancialAssistance::ApplicationController
     before_action :set_family
     before_action :find_application, :except => [:index, :index_with_filter, :new]
     if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
-      before_action :enable_bs4_layout, only: [:application_year_selection, :application_checklist, :edit, :eligibility_results, :review_and_submit, :review,
+      before_action :enable_bs4_layout, only: [:application_year_selection, :application_checklist, :edit, :eligibility_results, :review_and_submit, :review, :raw_application,
```

Comment:
raw_application is part of the admin flow not the consumer

Reply:
fixed